content/en/platform/corda/5.3/release-notes/5.2.md

```diff
@@ -0,0 +1,12 @@
+---
+description: "Enhancements, fixed issues, and known limitations for Corda 5.3."
+title: "Corda 5.3.0 Release Notes"
+date: 2024-01-10
+menu:
+  corda53:
+    identifier: corda53-release-notes
```

Comment:
```suggestion
    identifier: corda53-release-notes-530
```

(duplicate of content/en/platform/corda/5.3/release-notes/_index.md)

Should also rename file